docs/usage-as-operator.md

```diff
@@ -7,7 +7,8 @@ In this document we are describing how this configuration looks like for Azure a
 ## `CloudProfileConfig`
 
 The cloud profile configuration contains information about the update and failure domain counts in the Azure regions you want to offer.
-Additionally, it contains the real machine image identifiers in the Azure environment.
+Additionally, it contains the real machine image identifiers in the Azure environment. You can provide either URN for Azure Market Place images or id of [Shared Image Gallery](https://docs.microsoft.com/en-us/azure/virtual-machines/windows/shared-image-galleries) images.
```

Comment:
Please reference the linux docu instead of the windows one https://docs.microsoft.com/en-us/azure/virtual-machines/linux/shared-image-galleries

Reply:
Done.